app/models/food.rb

```diff
@@ -1,6 +1,7 @@
 class Food < ApplicationRecord
   belongs_to :user
   has_many :recipe_foods, dependent: :delete_all
+  has_many :foods, through: :recipe_foods
```

Comment:
This should be ```:recipes``` instead of ```:foods``` since we're in the Food model and want to create the relationship with the Recipe model